content/en/blog/2021/discovery-selectors/index.md

```diff
@@ -0,0 +1,85 @@
+* Inclusion and/or exclusion of namespaces, for example, all namespaces with label istio-discovery=enabled AND label key `app` equal to `helloworld`
+
+Note: `DiscoverySelectors` is not a security boundary. Istiod will continue to have access to all namespaces even when you have configured your `DiscoverySelectors`.
+
+## DiscoverySelectors in Action
+
+Assuming you know which namespaces to include as part of the service mesh, as a mesh administrator, you can configure `DiscoverySelectors` at installation time or post-installation by adding your desired discovery selectors to Istio’s MeshConfig resource. For example, you can configure Istio to discover only the namespaces that have the label `istio-discovery=enabled`.
+
+1. Using our examples earlier, let’s label the default namespace with label `istio-discovery=enabled`.
+
+    {{< text bash >}}
+    $ kubectl label namespace default istio-discovery=enabled
+    {{< /text >}}
+
+1. Use istioctl to apply the yaml with `DiscoverySelectors` to update your Istio installation. Note, to avoid any impact to your stable environment, we recommend that you use a different revision for your Istio installation:
+
+    {{< text bash >}}
+    $ istioctl install --skip-confirmation -f - <<EOF
+    apiVersion: install.istio.io/v1alpha1
+    kind: IstioOperator
+    metadata:
+    namespace: istio-system
+    spec:
+    # You may override parts of meshconfig by uncommenting the following lines.
+    meshConfig:
+      discoverySelectors:
+        - matchLabels:
+            istio-discovery: enabled
+    EOF
+    {{< /text >}}
+
+1. Display the endpoint configuration for the sleep pod:
+
+    {{< image link="./endpoints-with-discovery-selectors.png" caption="Endpoints for Sleep Deployment With DiscoverySelectors" >}}
+
+    Note this time the httpbin service in the ns-x namespace is NOT in the list of discovered endpoints, along with many other services that are not in the default namespace. If you display routes (or cluster or listeners) information for the sleep deployment, you will also notice much less configuration is returned:
+
+    {{< image link="./routes-with-discovery-selectors.png" caption="Routes for Sleep Deployment With DiscoverySelectors" >}}
+
+You can use `matchLabels` to configure multiple labels with AND semantics or use `matchLabels` sets to configure OR semantics among multiple labels. Whether you deploy services or pods to namespaces with different sets of labels or multiple application teams in your organization use different labeling conventions, `DiscoverySelectors` provides the flexibility you need. Furthermore, you could use `matchLabels` and `matchExpressions` together per our [documentation](https://github.com/istio/api/blob/master/mesh/v1alpha1/config.proto#L792). Refer to the Kubernetes [selector docs](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors) for additional detail on selector semantics.
+
+## DiscoverySelectors vs Sidecar Resource
```

Comment:
Are you saying `DiscoverySelectors` through out to (a) refer to a resource type (b) please the spelling checker?

It might be better to start saying "Discovery selectors" if that's the feature name.

Reply:
Yes that is the feature name.  I'll take your suggestion